packages/components/built/DecodeTransaction.d.ts

```diff
@@ -0,0 +1,4 @@
+export declare function DecodedransactionComponent(): import("react/jsx-runtime").JSX.Element;
```

Comment:
We should rename in all relevant files:
* `DecodedransactionComponent` -> `DecodeTransactionComponent`
* `Decodedransaction` -> `DecodeTransaction`